scripts/monitoring/cron-send-usage-pv.py

```diff
@@ -102,6 +102,42 @@ def get_pv_usage():
             capacity_int = int(capacity.strip('GIgi'))
             total = total + capacity_int
     return total
+def get_pv_usage_clusterresourcequota():
+    """get all the pv used if the cluster is using the clusterresourcequota"""
+    pv_info = runOCcmd_yaml(" get clusterresourcequota/persistent-volume ")
+
+
+    return pv_info
+
```

Comment:
please remove the extra whitespace (3 empy lines) above here, leave one empty line between this and the next function